content/en/about/community/toc/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Technical Oversight Committee
+description: Information describing the Istio technical oversight committee.
```

Comment:
Here TOC is not capitalized. It is capitalized in the text below.

Reply:
descriptiion as per website should be lower case should it not? If not, that's fine, I can modify it :)

Cheers,
-steve